src/frontend/js/containers/Login.js

```diff
@@ -0,0 +1,73 @@
+import Button from 'react-bootstrap/lib/Button';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login } from '../actions/Security';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { name: '', password: '' };
+  }
+
+  onChange(e, field) {
+    this.setState({ [field]: e.target.value });
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+    this.props.onSubmit(this.state);
+  }
+
+  render() {
+    const canSubmit = this.state.name.length > 3 && this.state.password.length > 2;
+    return <form className="login-form" onSubmit={this.onSubmit.bind(this)}>
+      <FormControl
+        disabled={this.props.loggingIn}
+        type="text"
+        onChange={e => this.onChange(e, 'name')}
+        placeholder="Family Name"
+        value={this.state.name} />
+      <FormControl
+        disabled={this.props.loggingIn}
+        type="password"
+        onChange={e => this.onChange(e, 'password')}
+        placeholder="Password"
+        value={this.state.password} />
+      <Button disabled={this.props.loggingIn || !canSubmit} type="submit">Login</Button>
+      {this.renderStatus()}
+    </form>;
+  }
+
+  renderStatus() {
+    if (this.props.loggingError !== '') {
```

Comment:
will `this.props.loggingError` ever be `null` and would you want to treat it differently? If not, then you could just do `if (!this.props.loggingError)` right?

Reply:
Right. I'll make that change and then no make that prop required.

Reply:
Actually, this needs to be `if (this.props.loggingError) {...`.